dtfs-central-api/src/helpers/amendments/tfm-facility-latest-completed-amendment-to-facility-value.ts

```diff
@@ -0,0 +1,25 @@
+import { TfmFacility } from '@ukef/dtfs2-common';
+import orderBy from 'lodash.orderby';
+import { filterCompletedAmendments } from './filter-completed-amendments';
+
+/**
+ * Get the latest completed facility value amendment
+ * @param tfmFacility - The tfm facility
+ * @returns The latest completed facility value amendment or null if there aren't any
+ */
+export const getLatestCompletedAmendmentToFacilityValue = (tfmFacility: TfmFacility): number | null => {
+  const { amendments } = tfmFacility;
+  if (!amendments) {
+    return null;
+  }
+
+  const completedAmendments = filterCompletedAmendments(amendments);
+
+  const latestAmendmentWithFacilityValue = orderBy(completedAmendments, ['updatedAt'], ['desc']).find(({ value }) => value !== undefined && value !== null);
+
+  if (latestAmendmentWithFacilityValue?.value !== undefined && latestAmendmentWithFacilityValue?.value !== null) {
```

Comment:
I changed it because I got worried about zero, which whilst I suspect it might not be possible to have a facility value of zero I'm not actually certain and so wanted to be careful given it is a number